src/streams/StreamItem.js

```diff
@@ -87,9 +88,15 @@ export default function StreamItem(props: Props) {
     backgroundColor !== undefined
       ? (foregroundColorFromBackground(backgroundColor): string)
       : themeColor;
+  const unreads =
+    unreadCount !== undefined && unreadCount > 0 ? `${unreadCount} unread messages` : '';
+  const accessibilityLabel = _('{name} {unreads}', {
```

Comment:
In the first argument passed to `_`, the things in curly braces aren't meant to include English labels about the data they represent. Here, I'm looking specifically at "unreads", which I see sometimes contains " unread messages". A translator won't see that label, and they won't be able to translate it. 🙂

To simplify a bit, let's include the unread count whenever it's present, even if it's zero. It doesn't hurt to know that there are zero unread messages in a stream, and if it turns out to be too verbose we can always change it.

Then, let's have the code make a choice between two expressions for the entire `accessibilityLabel`. I'm thinking something like this:

```js
  const accessibilityLabel =
    unreadCount === undefined
      ? _('Stream: {streamName}', {
          streamName: name,
        })
      : _('Stream: {streamName} ({unreadCount} unread messages)', {
          streamName: name,
          unreadCount,
        });
```

Then `'Stream: {streamName}'` and `'Stream: {streamName} ({unreadCount} unread messages)'` should go in messages_en.json, and I expect they'll be quite translatable. 🙂